### PR TITLE
Add Rainforest RAVEn integration

### PR DESCRIPTION
**Pull request by cottsay**
## Proposed change

Add basic documentation for the new `rainforest_raven` integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#80061
- Link to parent pull request in the Brands repository: home-assistant/brands#3762
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description.

File: source/_integrations/rainforest_raven.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Rainforest RAVEn
+description: Instructions on how to setup Rainforest RAVEn devices with Home Assistant.
+ha_category:
+  - Energy
+  - Sensor
+ha_release: 2022.12
```
Comment (frenck) [inline source/_integrations/rainforest_raven.markdown]:
```suggestion
ha_release: 2024.2
```

File: source/_integrations/rainforest_raven.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ha_codeowners:
+  - '@cottsay'
+ha_domain: rainforest_raven
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The RAVEn family of devices from Rainforest Automation, Inc. includes the [EMU-2](https://www.rainforestautomation.com/rfa-z105-2-emu-2-2/), a wireless device which reads energy usage information directly from your utility meter. This integration connects to a RAVEn-compatible device attached to the Home Assistant instance via USB to integrate instantaneous power demand, accumulated usage, and current unit pricing.
```
Comment (frenck) [inline source/_integrations/rainforest_raven.markdown]:
```suggestion
The RAVEn family of devices from Rainforest Automation, Inc. includes the [EMU-2](https://www.rainforestautomation.com/rfa-z105-2-emu-2-2/), a wireless device that reads energy usage information directly from your utility meter. This integration connects to a RAVEn-compatible device attached to the Home Assistant instance via USB to integrate instantaneous power demand, accumulated usage, and current unit pricing.
```

**Review by frenck — APPROVED** [review]
Thanks, @cottsay 👍 

../Frenck

**Pull request merged**